src/lib.rs

```diff
@@ -176,3 +184,48 @@ pub enum Generation {
     #[cfg(feature = "snp")]
     Genoa,
 }
+
+#[cfg(feature = "openssl")]
+impl From<Generation> for CertSevCaChain {
+    fn from(generation: Generation) -> CertSevCaChain {
+        use codicon::Decoder;
+
+        let (ark, ask) = match generation {
+            Generation::Naples => (SevBuiltin::naples::ARK, SevBuiltin::naples::ASK),
+            Generation::Rome => (SevBuiltin::rome::ARK, SevBuiltin::rome::ASK),
+            Generation::Milan => (SnpBuiltin::milan::ARK, SnpBuiltin::milan::ASK),
+            Generation::Genoa => (SnpBuiltin::genoa::ARK, SnpBuiltin::genoa::ASK),
```

Comment:
```
            Generation::Milan => (SnpBuiltin::milan::ARK, SnpBuiltin::milan::ASK),
            Generation::Genoa => (SnpBuiltin::genoa::ARK, SnpBuiltin::genoa::ASK),
```

We need to add legacy to certificates if we want to include SEV support for later generations.

Reply:
Can you elaborate?

Reply:
Fixed in the next commit.